config/yamlcfg/yaml2proto.go

```diff
@@ -28,15 +28,71 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+// getDefaults take all paths found through seeking, returns list of dirs with defaults
+func getDefaults(allPaths []string) (defaults []string, err error) {
+	dirs_found := make(map[string]bool)
+	for _, path := range allPaths {
+		if _, ok := map[string]bool{"default.yaml": true, "default.yml": true}[filepath.Base(path)]; ok {
```

Comment:
`if filepath.Base(path) == "default.yaml" or filepath.Base(path) == "default.yml"`, right?

I'm having trouble parsing this map literal or why it's there.

Reply:
Ah, map literal was us discovering a map can't be a constant, and deciding not to use a list instead. I agree the explicit OR makes more sense, done!